packages/api/src/middleware/psa.js

```diff
@@ -40,7 +52,11 @@ export function withPinningAuthorized(handler) {
   return async (event, ctx) => {
     // TODO: we need withAuth middleware so we don't have to do this twice
     const { user } = await validate(event, ctx)
-    const authorized = psaAllow.includes(String(user.id)) || psaAllow[0] === '*'
+
+    const authorized =
+      hasTag(user, 'PSA_ENABLED', 'true') &&
```

Comment:
we need to get this merged in: https://github.com/nftstorage/nft.storage/pull/1397